tests/Fixtures/Integration/misc/PHP8_0.test

```diff
@@ -30,7 +32,7 @@ $country = $session?->user?->getAddress()?->country;
 // https://wiki.php.net/rfc/shorter_attribute_syntax
 // https://wiki.php.net/rfc/shorter_attribute_syntax_change
 #[MyAttribute]
-#[\MyExample\MyAttribute]
+#[MyAttribute2]
```

Comment:
I believe that `#[MyAttribute]` and `#[\MyExample\MyAttribute]` was here to show that name conflict prevents the latter from being imported, so please keep it as it was before. If you need to import another attibute's FQCN, please add new attribute.